default.go

```diff
@@ -0,0 +1,27 @@
+package glutton
+
+import (
+	"io"
+	"log"
+	"net"
+)
+
+func handleDefault(conn net.Conn) error {
+	defer conn.Close()
+	log.Println("Handling connection with default handler...")
+	buf := make([]byte, 0, 81920)
+	tbuf := make([]byte, 1024)
+
+	for {
+		n, err := conn.Read(tbuf)
+		if err != nil {
+			if err != io.EOF {
+				log.Printf("Read error: %s", err)
+			}
+			break
+		}
+		buf = append(buf, tbuf[:n]...)
+	}
+	log.Printf("Data read: %q\n", string(buf))
+	return nil
+}
```

Comment:
Yes, obviously. In fact in my vision it will be the most impressive feature of glutton :)